app/models/cmr.rb

```diff
@@ -33,6 +33,14 @@ def self.cmr_request(url)
     end
   end
 
+  def self.get_raw_concept(concept_id, revision_id = nil)
```

Comment:
This code isn't necessary for this ticket but useful library I'm using for some side code that I use for small rake tasks, that I will eventually check in.